### PR TITLE
page 2023 added

### PR DESCRIPTION
**Pull request by Amyx000**

### CODE REVIEW

**Comment by atapas** [comment]
The PR should point to the 2-plays-a-month branch, not main.

File: components/Layout.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ const Layout = ({ links, children, title, description }) => {
   const [secondaryNavbar, setSecondaryNavbar] = useState(false);
 
   useEffect(() => {
-    if (currentPath === "/hackrplay/2022/home") {
+    if (currentPath === "/hackrplay/2022/home" || currentPath === "/hackrplay/2023/home") {
```
Comment (atapas) [inline components/Layout.js:34]: the 2023 event is not hackrplay. Please check the actual description of the issue.

File: pages/hackrplay/2023/home.js [hunk 2]
```diff
@@ -0,0 +1,208 @@
+import { useEffect } from "react";
```
Comment (atapas) [inline pages/hackrplay/2023/home.js:1]: Change the directory name and structure based on the suggestion in the description of the issue #109

**Review by atapas — CHANGES_REQUESTED** [review]

File: components/Layout.js (hunk 1 above)
Comment (Amyx000) [reply components/Layout.js:34]: yes got it

**Pull request closed without merging**